examples/templates/aws-linux/main.tf

```diff
@@ -93,6 +80,11 @@ Content-Disposition: attachment; filename="cloud-config.txt"
 #cloud-config
 cloud_final_modules:
 - [scripts-user, always]
+hostname: ${lower(data.coder_workspace.me.name)}
+users:
+- name: ${lower(data.coder_workspace.me.owner)}
```

Comment:
what restrictions do we have on coder usernames?  Is it a subset of the valid Linux usernames?

Reply:
I'm actually not sure here. I could change it to "coder" to be safe

Reply:
Ok, so a little googling: Linux usernames can have uppercase, lowercase, digits, hyphens and underscores -- we restrict our usernames to this minus the hyphens, so all good on character set.  However, Linux usernames can only be 32 characters long, at least in some flavors/system utilities and we allow unlimited.  I think it's fine to just truncate if the name is too long.

Reply:
Cool, thanks for looking into this. Usernames are not truncated, if longer than 32 characters using Terraform's `substr`. If usernames are shorter, nothing happens